vmdb/app/models/vm_or_template.rb

```diff
@@ -1165,7 +1165,7 @@ def self.refresh_ems(vm_ids)
   end
 
   def refresh_ems
-    raise "no #{ui_lookup(:table => "ext_management_systems")} or credentials defined" unless self.ext_management_system && self.ext_management_system.authentication_valid?
+    raise "no #{ui_lookup(:table => "ext_management_systems")} or credentials defined" unless self.ext_management_system && self.ext_management_system.has_credentials?
```

Comment:
authentication_check is responsible for populating the status with incomplete if we don't have credentials.  See here: https://github.com/jrafanie/manageiq/blob/82027846ff1aab1943cad86c97ffc1ff9c065513/vmdb/app/models/mixins/authentication_mixin.rb#L163

It should detect many situations such as no userid, bad password, unreachable, some other error:
https://github.com/jrafanie/manageiq/blob/82027846ff1aab1943cad86c97ffc1ff9c065513/vmdb/app/models/authentication.rb#L28